kuma/core/middleware.py

```diff
@@ -53,7 +54,7 @@ def process_request(self, request):
             return response
 
         request.path_info = '/' + prefixer.shortened_path
-        request.locale = prefixer.locale
+        request.locale = prefixer.locale or settings.LANGUAGE_CODE
```

Comment:
> What's the worst that could happen with this change?

I was hoping you had some ideas there. :grin: 

Paths that don't have a locale presumably don't need any locale-specific content. Things like links to attachments or `robots.txt`. But, I see `/admin` in that list also and I imagine Django probably looks at the `Accept-Language` header instead. But I'd have to dig if this affects that or bypasses it since it's in the ignore list.

I see 3 options:
1. Roll with this but dig deeper on the implications.
2. Merge the `request.LANGUAGE_CODE` change first assuming it will fix this.
3. Fix the tests since there are only really 3 tests that fail w/o this change.